tests/disp_dev/Makefile

```diff
@@ -0,0 +1,9 @@
+BOARD ?= stm32f429i-disc1
+include ../Makefile.tests_common
+
+DISABLE_MODULE += test_utils_interactive_sync
+
+USEMODULE += ili9341
+USEMODULE += disp_dev
+
```

Comment:
I guess this needs a blacklist for CI?

Reply:
A display can be wired to any board with SPI.

Reply:
So they can be compile tested, but why run the test script?

Reply:
There's no test script :)

Reply:
I see. So we avoid the issue (and convention: no new test application without test script).

Reply:
> So we avoid the issue (and convention: no new test application without test script)

It's the same for almost all driver. You can't automate the test unless the hardware is wired to the board.